tedpermission/src/main/java/com/gun0912/tedpermission/PermissionBuilder.java

```diff
@@ -2,43 +2,58 @@
 
 import android.content.Context;
 import android.content.Intent;
-import android.content.pm.ActivityInfo;
 import android.os.Build;
+import android.support.annotation.IntDef;
+import android.support.annotation.NonNull;
 import android.support.annotation.StringRes;
 
-import com.gun0912.tedpermission.util.ObjectUtils;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
+import static android.content.pm.ActivityInfo.*;
 
+@SuppressWarnings("unchecked")
```

Comment:
Is there any lint error about this?
I don't know why this should be added.

Reply:
For now there is no lint error about it. I just don't like compile warnings. Removed it anyway.

The only thing lint complains about right now is an absence of a korean translation. I gave the one "powered" by Google Translate. Probably I will need your assist in translating text because I'm not sure that the current translation is correct.